pkg/apis/config/v1alpha1/mesh_config.go

```diff
@@ -0,0 +1,64 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// MeshConfig is the configuration for the service mesh overall
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type MeshConfig struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec MeshConfigSpec `json:"spec,omitempty"`
+}
+
+// MeshConfigSpec is the spec for OSM's configuration
+type MeshConfigSpec struct {
+	Sidecar       SidecarSpec       `json:"sidecar,omitempty"`
+	Traffic       TrafficSpec       `json:"traffic,omitempty"`
+	Observability ObservabilitySpec `json:"observability,omitempty"`
+	Certificate   CertificateSpec   `json:"certificate,omitempty"`
+}
+
+// SidecarSpec is the spec for OSM's sidecar configuration
+type SidecarSpec struct {
+	EnablePrivilegedInitContainer bool   `json:"enable_privileged_init_container,omitempty"`
+	LogLevel                      string `json:"log_level,omitempty" default:"error"`
+}
+
+// TrafficSpec is the spec for OSM's traffic management configuration
+type TrafficSpec struct {
+	Egress                            bool   `json:"egress,omitempty"`
+	OutboundIPRangeExclusionList      string `json:"outbound_ip_range_exclusion_list,omitempty"`
```

Comment:
This should be a list of strings.
Please track this in a new issue and address in a subsequent PR.

Reply:
Updated in this PR